.github/workflows/analysis.yml

```diff
@@ -87,7 +87,6 @@ jobs:
   build_performance:
     runs-on: ubuntu-latest
     container: ghcr.io/acts-project/ubuntu2404:53
-    if: github.ref == 'refs/heads/main'
```

Comment:
Ah, I didn't realize this was here before. This job doesn't produce any output whatsoever on PRs (and right now it doesn't on main either).

Maybe we should disable the whole job altogether for now?

Reply:
Oh no that will now run on all PRs right? Should we directly fix this on main?

Reply:
Yeah I'll just add this back